apps/mobile/src/components/ChatDetailScreen/components/TextMessage.tsx

```diff
@@ -152,14 +165,32 @@ function TextMessage({
                   </Text>
                 </YStack>
               )}
-              <Text
-                selectable
-                fos={16}
-                fontFamily={'$body500'}
-                color={isMine ? '$black' : '$white'}
-              >
-                {message.message.text}
-              </Text>
+              {isExtended ? (
+                <TextInput
+                  caretHidden
+                  multiline
+                  hitSlop={{top: 15, bottom: 15, left: 10, right: 10}}
```

Comment:
Můžeme toto do constanty mimo funkci. Chtěl bych si být jistý, že to nebude triggrovat rerender. Better safe than sorry